Airports/EGGW/RMA.txt

```diff
@@ -1,15 +1,6 @@
-Luton RMA                 S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
-                          N052.06.08.000 W000.03.00.000 N052.00.51.001 W000.03.09.000
-                          N052.00.51.001 W000.03.09.000 N052.01.27.000 E000.00.00.001
-                          N052.01.27.000 E000.00.00.001 N051.51.46.000 E000.00.06.000
-                          N051.51.46.000 E000.00.06.000 N051.51.01.000 W000.00.25.000
-                          N051.51.01.000 W000.00.25.000 N051.44.09.000 W000.40.05.000
-                          N051.44.09.000 W000.40.05.000 N051.48.10.000 W000.41.55.000
-                          N051.48.10.000 W000.41.55.000 N051.46.59.002 W000.48.38.002
-                          N051.46.59.002 W000.48.38.002 N051.50.48.001 W000.49.26.000
-                          N051.50.48.001 W000.49.26.000 N051.53.56.000 W000.50.06.000
-                          N051.53.56.000 W000.50.06.000 N051.57.49.000 W000.40.48.000
-                          N051.57.49.000 W000.40.48.000 N052.03.16.000 W000.34.41.000
-                          N052.03.16.000 W000.34.41.000 N052.00.38.000 W000.28.32.000
-                          N052.00.38.000 W000.28.32.000 N052.06.06.000 W000.17.12.000
-                          N052.06.06.000 W000.17.12.000 N052.06.08.000 W000.03.00.000
+Luton RMA                  S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
In your picture of the RMA, the boundary line between Stansted Area A and Luton areas X and W doesn't extend down to the Luton CTA.

Reply:
Yes, this is intentional - reduces clutter, as the line can be derived by it being abeam LOREL and in line with the extended centreline